src/include/autoinstall/ask.rb

```diff
@@ -136,7 +176,7 @@ def askDialog
           title = Ops.get_string(ask, "title", "")
           back_label = Ops.get_string(ask, "back_label", back_label)
           ok_label = Ops.get_string(ask, "ok_label", ok_label)
-          timeout = Ops.get_integer(ask, "timeout", 0)
+          timeout = ask["timeout"] if ask.key?("timeout")
```

Comment:
This will reuse the timeout from the previous `Builtins.foreach` iteration.
Better remove the initialization at line 150 and use `ask.fetch("timeout", 0)` here.